src/shared/components/common/vote-buttons.tsx

```diff
@@ -114,53 +136,61 @@ export class VoteButtonsCompact extends Component<
         <button
           type="button"
           className={`btn btn-animate btn-sm btn-link py-0 px-1 ${
-            this.props.my_vote === 1 ? "text-info" : "text-muted"
+            this.props.myVote === 1 ? "text-info" : "text-muted"
           }`}
-          data-tippy-content={tippy(this.props.counts)}
+          data-tippy-content={tippy(
+            this.props.voteDisplayMode,
+            this.props.counts,
+          )}
           disabled={!UserService.Instance.myUserInfo}
           onClick={linkEvent(this, handleUpvote)}
           aria-label={I18NextService.i18n.t("upvote")}
-          aria-pressed={this.props.my_vote === 1}
+          aria-pressed={this.props.myVote === 1}
         >
           {this.state.upvoteLoading ? (
             <Spinner />
           ) : (
             <>
               <Icon icon="arrow-up1" classes="icon-inline small" />
-              {showScores() && (
-                <span className="ms-2">
-                  {numToSI(this.props.counts.upvotes)}
-                </span>
-              )}
+              {showScores() &&
+                this.props.voteContentType === VoteContentType.Post && (
+                  <span className="ms-2">
+                    {numToSI(this.props.counts.upvotes)}
+                  </span>
+                )}
             </>
           )}
         </button>
         {this.props.enableDownvotes && (
           <button
             type="button"
             className={`ms-2 btn btn-sm btn-link btn-animate btn py-0 px-1 ${
-              this.props.my_vote === -1 ? "text-danger" : "text-muted"
+              this.props.myVote === -1 ? "text-danger" : "text-muted"
             }`}
             disabled={!UserService.Instance.myUserInfo}
             onClick={linkEvent(this, handleDownvote)}
-            data-tippy-content={tippy(this.props.counts)}
+            data-tippy-content={tippy(
+              this.props.voteDisplayMode,
+              this.props.counts,
+            )}
             aria-label={I18NextService.i18n.t("downvote")}
-            aria-pressed={this.props.my_vote === -1}
+            aria-pressed={this.props.myVote === -1}
           >
             {this.state.downvoteLoading ? (
               <Spinner />
             ) : (
               <>
                 <Icon icon="arrow-down1" classes="icon-inline small" />
-                {showScores() && (
-                  <span
-                    className={classNames("ms-2", {
-                      invisible: this.props.counts.downvotes === 0,
-                    })}
-                  >
-                    {numToSI(this.props.counts.downvotes)}
-                  </span>
-                )}
+                {showScores() &&
+                  this.props.voteContentType === VoteContentType.Post && (
```

Comment:
For mobile post-listing view, it needs to show the vote counts on the buttons, because it doesn't have a header bar showing this info like comments. But its pointless to show these scores for comments, because its right in the header bar. I'm trying to reduce the amount of information overload and duped info everywhere possible.